token/jwt/claims_jwt.go

```diff
@@ -102,7 +122,13 @@ func (c *JWTClaims) ToMap() map[string]interface{} {
 	ret["exp"] = float64(c.ExpiresAt.Unix()) // jwt-go does not support int64 as datatype
 
 	if c.Scope != nil {
-		ret["scp"] = c.Scope
+		// ScopeField default (when value is JWTScopeFieldUnset) is the list for backwards compatibility with old versions of fosite.
```

Comment:
@aeneasr Are you sure we want to keep backwards compatibility here? Existing behavior is not by the spec and it can be surprised that JWT tokens looks like they do not have scopes (because you are looking at the standard field). Should we change default here to be by the spec by default?